__main__.py

```diff
@@ -78,6 +78,38 @@ def main():
             reset_retriever(args.scope)
             return
 
+        elif args.command == 'new_json':
+            # create new JSON script
+            create_json()
+            return
+
+        elif args.command == 'edit_json':
+            # edit existing JSON script
+            if args.dataset is None:
```

Comment:
we can use `argparse` in [get_opts.py](https://github.com/weecology/retriever/blob/master/lib/get_opts.py) to make sure `edit_json` has a dataset argument.